src/lib/SX127xDriver/SX127x.cpp

```diff
@@ -340,7 +354,7 @@ bool SX127xDriver::DetectChip(SX12XX_Radio_Number_t radioNumber)
   }
   else
   {
-    DBGLN(" found! (match by REG_VERSION == 0x12)");
+    DBGLN(" found! (match by REG_VERSION == SX127X_VERSION)");
```

Comment:
Does this work?  I think it needs to be something like below.

`DBGLN(" found! (match by REG_VERSION == 0x%x"", SX127X_VERSION);`

Reply:
Kind-of since I was being lazy, and it'll just print SX127X_VERSION which a dev could look up. I'll fix it so it's more obvious when someone is debugging

Reply:
:neutral_face:

Reply:
Fixed